multimodal/tests/unittests/predictor/test_predictor_model_dump.py

```diff
@@ -107,8 +107,7 @@ def test_fusion_model_dump():
         time_limit=5,
         seed=42,
     )
-    predictor.dump_hf_text(path=model_dump_path)
-    predictor.dump_timm_image(path=model_dump_path)
+    predictor.dump_multimodal(path=model_dump_path)
```

Comment:
Is there a way to dump only the `hf_text` branch of the model?

Reply:
User can always select the branch based on model's prefix. We can add a branch selection parameter here but is there a reason that we only want to dump a (few) specific branch(es)?